bin/pcm-add-extra-masks

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+
+def main(
+    *,
+    mask_config_file,
+    input_mask_file,
+    output_mask_file,
+    output_masked_pixels_file,
+):
+    import json
+    import healsparse as hsp
+
+    with open(mask_config_file, 'r') as config_file:
+        mask_config = json.load(config_file)
+
+    masked_pixels = hsp.HealSparseMap.make_empty(
+        32,
+        2**14,
+        bool,
+        sentinel=False,
+        bit_packed=True,
+    )
+
+    new_hsm = hsp.HealSparseMap.read(input_mask_file)
+
+    for obj, data in mask_config.items():
+        ra = data["ra"]
+        dec = data["dec"]
+        semi_major = data["semi_major"]
+        semi_minor = data["semi_minor"]
+        pa = data["pa"]
+
+        nel = hsp.geom.Ellipse(
+            ra=ra,
+            dec=dec,
+            semi_major=semi_major,
+            semi_minor=semi_minor,
+            alpha=pa,
+            value=1,
+        )
+
+        hnel16 = nel.get_map(nside_coverage=32, nside_sparse=2**14, dtype=int)
+        masked_pixels.update_values_pix(hnel16.valid_pixels, True)
```

Comment:
It makes me unhappy that you have to do this, so I'm writing new `hpgeom` and `healsparse` code to make this easier.  But right now this is what you need.